test/test_KF/main.cpp

```diff
@@ -54,10 +56,14 @@ class KFSimulation : public GaussianFilter
 };
 
 
-int main()
+int main(int argc, char* argv[])
 {
     std::cout << "Running a KF filter on a simulated target." << std::endl;
-    std::cout << "Data is logged in the test folder with prefix testKF." << std::endl;
+
+    const bool write_to_file = (argc > 1 ? std::string(argv[1]) == "ON" : false);
+    if (write_to_file)
+        std::cout << "Data is logged in the test folder with prefix testSIS." << std::endl;
```

Comment:
I think it should be `... with prefix testKF`.

Reply:
Done!